presto-iceberg/src/test/java/com/facebook/presto/iceberg/TestOutputColumnTypes.java

```diff
@@ -127,15 +132,15 @@ public void testOutputColumnsForCreateTableAS()
 
         assertThat(event.getIoMetadata().getOutput().get().getColumns().get())
                 .containsExactly(
-                        new Column("orderkey", "bigint"),
-                        new Column("custkey", "bigint"),
-                        new Column("orderstatus", "varchar"),
-                        new Column("totalprice", "double"),
-                        new Column("orderdate", "date"),
-                        new Column("orderpriority", "varchar"),
-                        new Column("clerk", "varchar"),
-                        new Column("shippriority", "integer"),
-                        new Column("comment", "varchar"));
+                        new OutputColumnMetadata("orderkey", "bigint", ImmutableSet.of()),
+                        new OutputColumnMetadata("custkey", "bigint", ImmutableSet.of()),
+                        new OutputColumnMetadata("orderstatus", "varchar", ImmutableSet.of()),
+                        new OutputColumnMetadata("totalprice", "double", ImmutableSet.of()),
+                        new OutputColumnMetadata("orderdate", "date", ImmutableSet.of()),
+                        new OutputColumnMetadata("orderpriority", "varchar", ImmutableSet.of()),
+                        new OutputColumnMetadata("clerk", "varchar", ImmutableSet.of()),
+                        new OutputColumnMetadata("shippriority", "integer", ImmutableSet.of()),
+                        new OutputColumnMetadata("comment", "varchar", ImmutableSet.of()));
```

Comment:
Why do we have empty sourceColumns set here? Shouldn't these have the source columns just like the previous test case?